apps/pie-microsite/src/__tests__/_11ty/shortcodes/list.test.js

```diff
@@ -0,0 +1,16 @@
+const list = require('../../../_11ty/shortcodes/list');
+
+describe('list.js', () => {
+    const items = ['a', 'b', 'c', 'd'];
+
+    it.each([
+        'ordered',
+        'pill'
+    ])('should return the expected HTML', type => {
+        // act
+        const result = list({ type, items });
+
+        // assert
+        expect(result).toMatchSnapshot();
+    });
```

Comment:
Would be good to include a test(s) that checks the different types you can pass in